j2objc.gradle

```
@@ -417,6 +421,121 @@ class J2objcTranslateTask extends DefaultTask {
+                executable binary
+                args "-I${srcDir}"
+                args "-c"
+
+                args flagMap[arch]
+                args "${project.j2objcConfig.compileFlags}".split()
+
+                def srcFiles = project.files(project.fileTree(
+                    dir: srcDir, includes: ["**/*.mm",  "**/*.m"]))
+                srcFiles.each { file ->
+                    args file.path
+                }
+            }
+
+            // Create arch specific archives 
+            println "Building ${arch} archive."           
+            project.exec {
+                workingDir "${archDir.path}"
+                commandLine "ar"
+                args  "rcs"
+                args  "${project.j2objcConfig.staticLibName}"
+
+                def srcFiles = project.files(project.fileTree(
+                    dir: archDir, includes: ["**/*.o"]))
+                srcFiles.each { file ->
+                    args file.path
+                }
+            }
+        }
+
+        def distDir = new File("${project.j2objcConfig.distributionDir}")
+        def includeDir = new File(distDir, "include")
+        def libDir = new File(distDir, "lib")
+        if (!includeDir.exists()) {
+            includeDir.mkdirs()
+        }
+        if (!libDir.exists()) {
+            libDir.mkdirs()
+        }
+
+        println "Copying header files to ${includeDir.path}"
+        project.copy {
+            includeEmptyDirs = false
+            from destDir
+            into includeDir
+            include "**/*.h"
+        }
+
+        println "Merging Universal archive to ${libDir.path}"   
+        project.exec {
+            workingDir "${libDir.path}"
+            commandLine "lipo"
+            args "-create"
+
+            for (arch in flagMap.keySet()) {
+                def archDir = new File(destDir, arch)
+                def archLib = new File(archDir, "${project.j2objcConfig.staticLibName}" )
+                if (archLib.exists()) {
```

If archLib doesn't exist then it should fail... as that indicates something that should've been built but wasn't. Don't hide the error by ignoring it if it doesn't exist.